apps/teams-test-app/src/components/utils/ApiWithTextInput.tsx

```diff
@@ -21,7 +21,7 @@ export interface ApiWithTextInputProps<T> {
   defaultInput?: string;
 }
 
-export const ApiWithTextInput = <T extends unknown>(props: ApiWithTextInputProps<T>): React.ReactElement => {
+export const ApiWithTextInput = <T,>(props: ApiWithTextInputProps<T>): React.ReactElement => {
```

Comment:
>, [](http://example.com/codeflow?start=34&length=1)

The type parameter needs to be declared with a comma here because this is a TSX file (which has ambiguous angle bracket parsing in some contexts, where it can't tell if you're reference an HTML element/control or if you're using a generic). In this case, it would have been ambiguous so the comma is required to indicate that this is a type parameter.

Reply:
Thanks for the explanation, I had no idea!